draft-ietf-masque-connect-ip.md

```diff
@@ -1331,6 +1331,37 @@ and outer IP headers does not apply because the outer connection will react
 correctly to congestion notifications if it uses ECN. The inner traffic can
 also use ECN, independently of whether it is in use on the outer connection.
 
+## Diffserv Considerations
+
+The tunneled IP traffic can have a Diffserv Code Point (DSCP) {{!DSCP=RFC2474}}
+set in the traffic class IP header field to request a particular per hop
+behavior. The client or IP proxy handling an IP proxying request if part of the
+differv domain and configured with the domains mapping MAY implement traffic
```

Comment:
```suggestion
behavior. If a client or IP proxy handling an IP proxying request is configured as part
of a diffserv domain, it MAY implement traffic
```